src/Microsoft.XmlSerializer.Generator/src/Sgen.cs

```diff
@@ -418,16 +426,16 @@ private void WriteHeader()
         private void WriteHelp()
         {
             Console.Out.WriteLine(SR.Format(SR.HelpDescription));
-            Console.Out.WriteLine(SR.Format(SR.HelpUsage, this.GetType().Assembly.GetName().Name));
+            Console.Out.WriteLine(SR.Format(SR.HelpUsage, this.GetType().Assembly.GetName().Name.Substring(7)));
             Console.Out.WriteLine(SR.Format(SR.HelpDevOptions));
-            Console.Out.WriteLine(SR.Format(SR.HelpAssembly, "/assembly:", "/a:"));
-            Console.Out.WriteLine(SR.Format(SR.HelpType, "/type:", "/t:"));
-            Console.Out.WriteLine(SR.Format(SR.HelpProxy, "/proxytypes", "/p"));
-            Console.Out.WriteLine(SR.Format(SR.HelpForce, "/force", "/f"));
-            Console.Out.WriteLine(SR.Format(SR.HelpOut, "/out:", "/o:"));
+            Console.Out.WriteLine(SR.Format(SR.HelpAssembly, "--assembly:", "-a:"));
+            Console.Out.WriteLine(SR.Format(SR.HelpType, "--type:"));
+            Console.Out.WriteLine(SR.Format(SR.HelpProxy, "--proxytypes"));
+            Console.Out.WriteLine(SR.Format(SR.HelpForce, "--force"));
+            Console.Out.WriteLine(SR.Format(SR.HelpOut, "--out:", "-o:"));
 
             Console.Out.WriteLine(SR.Format(SR.HelpMiscOptions));
-            Console.Out.WriteLine(SR.Format(SR.HelpHelp, "/?", "/help"));
+            Console.Out.WriteLine(SR.Format(SR.HelpHelp, "--help", "-h"));
```

Comment:
Can you please make sure the output is something like this? Please check out the formating like "dotnet -h".
```
  -h|--help             Show help.
```